api/routes/validation/customerValidation.js

```diff
@@ -32,30 +34,29 @@ class CustomerValidation {
     }
 
     static ValidateAllFieldsUndefined(customerData) {
```

Comment:
I think that name of this function might be confusing a bit.
Maybe something like ValidateUndefinedFields or just ValidateUndefined?